platform/ios/docs/guides/Working with Style Functions.md

```diff
@@ -4,15 +4,15 @@
   Edit platform/darwin/scripts/generate-style-code.js, then run `make darwin-style-code`.
 -->
 
-# Data-Driven Styling
+# Working with Style Functions
 
-Mapbox’s data-driven styling features allow you to use attributes in the data to style your maps. You can style map features automatically based on their individual attributes.
+Mapbox’s runtime and data-driven styling features allow you to style your maps as a function of the current map zoom level, attributes in the map source data, or both. With data-driven styling, you can style map features automatically based on their individual attributes.
```

Comment:
Now that this article is specifically about an API (style functions) rather than a feature (data-driven styling), we should refocus the introduction around the style functions themselves. Here’s an introduction adapted from “Information for Style Authors”:

> [Runtime styling](https://www.mapbox.com/ios-sdk/api/3.5.0/runtime-styling.html) enables you to modify every aspect of the map’s appearance dynamically as a user interacts with your application. Much of the runtime styling API allows you to specify _style functions_ instead of constant values. A style function allows you to specify in advance how a layout or paint attribute will vary as the zoom level changes or how the appearance of individual features varies based on metadata provided by a content source.